README.rst

```diff
@@ -81,4 +87,10 @@ References
 
 [7] Harastani M, Eltsov M, Leforestier A, Jonic S: TomoFlow: Analysis of continuous conformational variability of macromolecules in cryogenic subtomograms based on 3D dense optical flow. J Mol Biol 2021,167381. `[Author’s version] <https://hal.archives-ouvertes.fr/hal-03452809>`__ `[Journal] <https://doi.org/10.1016/j.jmb.2021.167381>`__
 
```

Comment:
I believe these two references are not good at the plugin readme. They can be cited when we run an NMMD protocol.